### PR TITLE
Manual Docs 3

### PR DESCRIPTION
**Pull request by misteroneill**
Moved the "removing players" guide to "player workflows" guide.

## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: docs/guides/player-workflows.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Player Workflows
+
```
Comment (brandonocasey) [inline docs/guides/player-workflows.md:2]: Should this file contain other advanced workflows? like using `ready` or maybe using other events on the player to do things.

**Review by brandonocasey — CHANGES_REQUESTED** [review]

File: docs/guides/player-workflows.md (hunk 1 above)
Comment (gkatsev) [reply docs/guides/player-workflows.md:2]: We should add a story to writeup a simple way of using videojs with React.

File: docs/guides/player-workflows.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+1. Trigger a `"dispose"` event on the player, allowing for any custom cleanup tasks that need to be run by your integration.
+1. Remove all event listeners from the player.
+1. Remove the player's DOM element(s).
+
+Additionally, these actions are recursively applied to _all_ the player's child components.
+
+> **Note**: Do _not_ remove players via standard DOM removal methods: this will leave listeners and other objects in memory that you might not be able to clean up!
+
+### Signs of an Undisposed Player
```
Comment (gkatsev) [inline docs/guides/player-workflows.md:31]: This reminds me, we should add the `vdata123124` and this as a question in the FAQ.

Comment (gkatsev) [reply docs/guides/player-workflows.md:31]: The vdata error is a similar thing where the element was removed from the DOM but an event triggered for it.

File: docs/guides/player-workflows.md (hunk 1 above)
Comment (gkatsev) [reply docs/guides/player-workflows.md:2]: Looks like this was originally a "removing players" document.

Comment (brandonocasey) [reply docs/guides/player-workflows.md:2]: it might be a good idea to make an issue for this just in case

Comment (misteroneill) [reply docs/guides/player-workflows.md:2]: Yeah, this was the "removing players" document. I think it makes sense to have a document devoted to "doing things with players other than creating them". That's the intent of this.

File: docs/guides/player-workflows.md (hunk 2 above)
Comment (misteroneill) [reply docs/guides/player-workflows.md:31]: I'll mention both.

File: docs/guides/player-workflows.md (hunk 1 above)
Comment (misteroneill) [reply docs/guides/player-workflows.md:2]: `ready` is covered in the setup guide, I believe.

Comment (misteroneill) [reply docs/guides/player-workflows.md:2]: And I added a section at the bottom for "Using Video.js with..." and empty sub-headings for jQuery, React, Ember, and Angular.

File: docs/guides/player-workflows.md (hunk 2 above)
Comment (brandonocasey) [reply docs/guides/player-workflows.md:31]: vdata was added to the troubleshooting/faq guides

**Review by brandonocasey — APPROVED** [review]

**Pull request closed without merging**